src/common/bidi/Browser.ts

```diff
@@ -0,0 +1,49 @@
+import * as BrowserApi from '../../api/Browser.js';
+import {Connection} from './Connection.js';
+import {ChildProcess} from 'child_process';
+
+/**
+ * @internal
+ */
+export class Browser extends BrowserApi.Browser {
```

Comment:
yeah, it's already scoped by the folder name so there is no need for a prefix, imo

Reply:
But it ends up with the constructions like `import {Connection as BiDiConnection} from '../common/bidi/Connection.js';`.

Reply:
Yeah, true, but I don't think it's critical and it can be changed at any point later. We also plan eventually to only support BiDi so there will be only a single Browser/Connection etc.

Reply:
SGTM